README.md

```diff
@@ -71,38 +42,74 @@ options = ExecutionParameters(nshots=1000)
 results = platform.execute([sequence], options)
```

Comment:
If #1052 is approved:
```suggestion
results = platform.execute([sequence], nshots=1000)
```

Reply:
Done in ee0b9d07478a45dd9a1ab159bb8d1f82f4d431ec, thanks.